src/store/actions.ts

```diff
@@ -299,5 +300,37 @@ export default {
 
   setRestorationExpiry ({ commit }, expiry: string) {
     commit('mutateRestorationExpiry', expiry)
+  },
+
+  setRestorationApprovalType ({ commit }, type: ApprovalTypes) {
+    commit('mutateRestorationApprovalType', type)
+  },
```

Comment:
Would it be possible to remove this action method and call the mutation directly?  I know you're following the pattern set previously, but there's no reason to call an action when calling mutation will suffice.   Actions are typically for async operations and anything that includes business logic.  Mutations should be used when you're simply changing state -- instantaneous operations.

Reply:
So you mean in my code instead of calling `setRestorationApprovalType`, I would import and call `mutateRestorationApprovalType`? I can then remove all these new setters?

Reply:
Yes, correct

Reply:
Looks great! I'm OK with this conversation (but it's not mine to resolve).